src/go/rpk/pkg/cli/topic/add_partitions.go

```diff
@@ -65,11 +66,17 @@ func newAddPartitionsCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 
 			for _, resp := range resps.Sorted() {
 				msg := "OK"
+				if resp.ErrMessage != "" {
+					zap.L().Sugar().Debugf("redpanda returned error message: %v", resp.ErrMessage)
+				}
 				if e := resp.Err; e != nil {
 					if errors.Is(e, kerr.InvalidPartitions) && num > 0 {
 						msg = fmt.Sprintf("INVALID_PARTITIONS: unable to add %d partitions due to hardware constraints", num)
 					} else {
 						msg = e.Error()
+						if resp.ErrMessage != "" {
+							msg += ": " + resp.ErrMessage
```

Comment:
I think instead we want to change this to
```go
if ke := (*kerr.Error)(nil); errors.As(e, ke) {
```
and then use ke.Message + ":" + resp.ErrMessage

As is, this will say something like: "INVALID_REPLICATION_FACTOR: standard message: detailed message", and we can nix the standard message part.